src/ammer/stub/StubLua.hx

```diff
@@ -39,7 +39,7 @@ class StubLua {
       case Bool: 'lua_pushboolean(L, $expr)';
       case Int: 'lua_pushinteger(L, $expr)';
       case Float: 'lua_pushnumber(L, $expr)';
-      case String | Bytes if (size != null): 'lua_pushlstring(L, $expr, $size)';
+      case WithSize(_,String|Bytes) : 'lua_pushlstring(L, $expr, $size)';
```

Comment:
There's this "WithSize" enum that seems to contain a size argument as an expression.  There wasn't an explicit case for it and it was erroring out.  The WithSize enum contains an expr representing its size, but the size is already passed in to the function (as an optional argument).  It seems a little flakey, I was wondering if we could rely more on the special enum than the optional argument.